wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java

```diff
@@ -206,4 +207,20 @@ private static boolean isTextGenerationModel(LmiUtils.HuggingFaceModelConfig mod
                 OPTIMIZED_TASK_ARCHITECTURES);
         return false;
     }
+
+    private static void setPythonExecutable(Properties lmiProperties) {
+        if (lmiProperties.containsKey("option.pythonExecutable")) {
+            return;
+        }
+        String rollingBatch = lmiProperties.getProperty("option.rolling_batch");
+        if ("vllm".equals(rollingBatch)) {
```

Comment:
How about option.rolling_batch=auto?

Reply:
we translate rolling_batch=auto earlier in this process in setRollingBatch - we shouldn't have auto as a value here